use hyphens in class file names. closes #1.

codeimpossible opened the pull request.

codeimpossible committed:
use hyphens in class file names. closes #1.

jhofker commented on README.md:
````diff
@@ -1715,7 +1715,7 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
+    import CheckBox from './check-box';
     ```
 
   - [22.7](#22.7) <a name='22.7'></a> Use camelCase when you export-default a function. Your filename should be identical to your function's name.
````
Fix here, too? :smiley:

codeimpossible replied:
yep.

Briankaiser commented:
Is there a reason for this change? It seems kind of nice to have the filename exactly match the class name?

codeimpossible committed:
update name convention for exported functions

files that contain a single function should follow the hyphenated file naming convention.

codeimpossible commented:
@Briankaiser Good point. IMHO the hyphens are easier to read. Also, it matches our current standards.

blainekasten commented:
This also goes against #5. That PR says that file names should PascalCased

blainekasten commented:
So what do we do here?
